app/views/certificates/index.html.slim

```diff
@@ -4,8 +4,8 @@ h1= t('.volunteers_certificates', name: @volunteer.contact.full_name)
   table.table
     thead
       tr
-        th= t('created_at')
-        th= t('updated_at')
+        th= t_attr(:created_at, Certificate)
```

Comment:
`created_at` and `updated_at` are included in the `generic_keys` so you can just have `t_attr(:created_at)` (and remove the individual translations).